### DISCUSSION TITLE
(Multi)Splitter, calculated initial size

### DISCUSSION THREAD

**Discussion by ianclough** [microsoft/fluentui-blazor#1761] [General]
Both Splitter and MultiSplitter have and OnResize event which supplies the new calculated panel sizes. It would be useful to also have a way of retrieving the initial calculated panel sizes. I'm not sure if this would be possible without support from the underlying Fluent UI Web Components.

**Comment by dvoituron** [comment]
The MultiSplitter event give you the pourcentage of panels. I don't think you have a pixel size. So the initial is always 100%. Is it correct ?

**Reply by ianclough** [reply]
Ok thank you, I hadn't realised the MultiSplitter reported the size is percentages as I haven't actually tried this component yet. I intend to switch to the MultiSplitter very soon for my IDE like application.
